### ISSUE TITLE
Feature request - Alternative Startup Modes

### ISSUE THREAD

**Issue by n6smith** [VCVRack/Rack#1147]
Would it be possible to have some kind of alternative start up modes for VCV rack via a key combo so you could launch without loading the last Autosave but instead launch using either the saved Template and/or a Blank Rack. This would be helpful when having issues with an existing patch that keeps reloading on VCV Launch (Aka safe mode launch) or when you don't want to load an Autosave patch that, for example, uses Host that is launching and loading large sample libraries that you must wait to load before loading a new template or fresh patch.

**Comment by rubyglow** [comment]
(that was Nigel above)
Second this feature request! Trying to flesh it out a bit....

The simple version:
- If the Control key is held when Rack launches, it is detected, and Rack loads to a blank slate, without loading autosave or template.

Or the luxury version:
- If the Control key is held when Rack launches, it is detected, and Rack loads to a blank slate, without loading autosave or template.
- If the Alt key is held when Rack launches, it is detected, and Rack loads the template, if present, rather than the autosave, and if no template is present it loads a blank slate.

It'd be very nice to get something like this in for 1.0 :-)
Thanks!

**Comment by aptrn** [comment]
Different executables might be a thing to, similar to "Reaper (create new project)" from Start Menu

**Comment by rubyglow** [comment]
Right. I'm not sure how well alternate program shortcuts work on Mac though, and on Linux I think all bets are off. So (also) having the keyboard option would be more universal.

**Comment by gsynuh** [comment]
I like the idea, but I feel I'd forget about it too often...
I'd like to add to the discussion with these suggestions :

1. add an option to force load autosaves or a specific template in a preferences.txt files

2. a floating window "launcher" to choose between starting empty, with an autosave, or a list of recent files (that launcher could also be disabled in the preferences file for people feeling this is an unwanted extra step)

**Comment by jponsford** [comment]
Just playing around with command-line options, it seems like 'rack --help' will start up with an empty patch; and then the next time rack is run, it loads the last Autosave? [NB: This is simply anecdotal observation on my system (win10).]

**Comment by catronomix** [comment]
I think a simpler solution would be to have a command line switch for loading a specific patch and then make a few different shortcuts, this way it stays the same for all platforms.

**Comment by AndrewBelt** [comment]
@catronomix That is already present.

**Comment by catronomix** [comment]
AH voila problemo solved. :)
I hope after v1 when all the features and most modules have settled down someone will combine and structurize all online informationinto an actual paper 'VCV bible'

**Comment by AndrewBelt** [comment]
This already exists: https://vcvrack.com/manual/Installing.html#command-line-usage

**Comment by catronomix** [comment]
That one is missing descriptions of all the modules in the plugin manager, tips and trick on how to patch things up, foldable edges and that nice book smell :p
